### ISSUE TITLE
Weekday names `:wide` format is the same as `:abbreviated`

### ISSUE THREAD

**Issue by DRBragg** [twitter/twitter-cldr-rb#245]
**Describe the bug**
`TwitterCldr::Shared::Calendar.new(:en).weekdays(:wide)` used to return `{:fri=>"Friday", :mon=>"Monday", :sat=>"Saturday", :sun=>"Sunday", :thu=>"Thursday", :tue=>"Tuesday", :wed=>"Wednesday"}` but after upgrading from `6.4.0` to `6.9.0` it returns `{:fri=>"Fri", :mon=>"Mon", :sat=>"Sat", :sun=>"Sun", :thu=>"Thu", :tue=>"Tue", :wed=>"Wed"}`

**To Reproduce**
1. Load `6.4.0` in irb
2. Run `TwitterCldr::Shared::Calendar.new(:en).weekdays(:wide)`
3. Note the output
4. Load `6.9.0` in irb
5. Run `TwitterCldr::Shared::Calendar.new(:en).weekdays(:wide)`
6. Compare the output
7. Cry...

**Expected behavior**
`TwitterCldr::Shared::Calendar.new(:en).weekdays(:wide)` should return `{:fri=>"Friday", :mon=>"Monday", :sat=>"Saturday", :sun=>"Sunday", :thu=>"Thursday", :tue=>"Tuesday", :wed=>"Wednesday"}`

**Screenshots**

**Environment**
Rails 4.2.11.3, Ruby 2.4.10

**Additional context**
I'm running out to Hockey practice now but I'll check it on 6.6.0 when I get home.  Sorry for the rushed bug submission.

**Comment by camertron** [comment]
Thanks for the bug report @DRBragg :) I see what happened here. It has to do with a change I made along with the upgrade to CLDR v40 that tries to resolve these CLDR alias things earlier in the data generation process. Unfortunately data from the root locale is winning out here when it shouldn't be. I've got a fix in the works.

**Comment by camertron** [comment]
Hey @DRBragg, following up on this. The fix has proven more difficult than anticipated, just wanted you to know I haven't forgotten about it :)

**Comment by camertron** [comment]
Alright, a little more context here. After digging through the actual CLDR data, it looks like the lib was actually doing the wrong thing in all versions previous to v6.8.0. In other words, v6.8.0 is actually doing the _right_ thing, but it now appears wrong because of how the lib used to work. This is the kind of thing people are talking about when they say "it's not a bug, it's a feature!"

The actual fix here is to change the default calendar format to (confusingly) "format" instead of "stand-alone." It's possible to get the right return value by passing `:format` as the second argument to `#weekdays`, but it's pretty surprising behavior without it.

Turns out a bunch of the work I did to correctly resolve aliases in the CLDR data wasn't necessary, but did result in more correct data for a bunch of locales. So 🤷
